### DISCUSSION TITLE
absence calender

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/Paid-plugins#177] [Questions & Answers] [answered]
We would like to have a absence calendar, where workers can see, when their co-worker are absent (holiday, sick). Is there something like this available? I see only the calender with worked hours, and only for one user each.

**Comment by kevinpapst** [comment, marked as answer]
No. But something like that is already planned for the WorkContract plugin.

**Comment by sherrlich** [comment]
This feature would be great. 
The team lead will get an overview and does not need to maintain a second list, where all the absence are tracked to avoid absence conflicts/collisions with other team members

**Comment by pkoevesdi** [comment]
Any news on this? Is this being worked on? Because our custom solution - a one way sync to outlook calendar - fails from time to time and is hard to maintain. So everytime ist fails, I wonder, when Kimai will provide an absence calendar, preferably as an ics-Link.
